docs/how-to-guides/develop-a-prompty/index.md

```diff
@@ -37,11 +39,19 @@ model:
   parameters:
     max_tokens: 128
     temperature: 0.2
+inputs:
+  first_name:
+    type: string
+    default: John
```

Comment:
remove default, let's only discuss this in our future doc describe signature part

Reply:
remove default